src/NzbDrone.Core/MediaFiles/MediaInfo/MediaInfoFormatter.cs

```diff
@@ -71,6 +72,26 @@ public static string FormatAudioCodec(MediaInfoModel mediaInfo, string sceneName
 
             if (audioFormat.EqualsIgnoreCase("DTS"))
             {
+                if (audioAdditionalFeatures.StartsWithIgnoreCase("XLL"))
+                {
+                    if (audioAdditionalFeatures.EndsWithIgnoreCase("X"))
+                    {
+                        return "DTS-X";
+                    }
+
+                    return "DTS-HD MA";
```

Comment:
I was thinking that too. But if I don't forget I'll poll some users about it. Otherwise we'll leave it as it till we get the first complaint. :smile:

Reply:
I got like one response when I asked about it, and that user thought that most people would prefer the longer one since it's more accurate. So let's keep the current token.